.github/workflows/managed.yml

```diff
@@ -68,21 +68,21 @@ jobs:
       - name: Restore Go build cache
         if: ${{ fromJSON(env.DEVCONTAINER_CACHE_ENABLED) }}
         continue-on-error: true
-        run: docker cp ~/.cache/go-build pmm-managed-server:/root/.cache/go-build
+        run: docker cp ~/.cache/go-build pmm-server:/root/.cache/go-build
```

Comment:
Can't we just mount this directory and the one below to the docker container instead of copying?

Reply:
Probably, but not in the scope of this PR. Also now these caches are disabled.